### PR TITLE
docs: fix and improve the description of the default reveal width

### LINKED ISSUE #8707: "Standard" Reveal width doesn't match docs (Foundation 6)
**How can we reproduce this bug?**

1. Build a Reveal modal using Foundation 6
2. View on a medium or larger screen width
3. See fixed 600px dialog

**What did you expect to happen?**

According to http://foundation.zurb.com/sites/docs/reveal.html#sizing the width of a medium+ Reveal dialog should be 80%.  This is what I expected coming from Foundation 5 as well.

**What happened instead?**

The width is set to 600px.

Either the documentation or the code is wrong.  I'm inclined to say it's the code that needs to be fixed.

### PR DESCRIPTION
**Pull request by DanielRuf**
This provides a better description of the default width of the reveal.

Closes #8707

### CODE REVIEW

File: docs/pages/reveal.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ You can use our handy [close button](close-button.html) styles to do this:
 
 ## Sizing
 
-On small screens, a modal is always 100% of the width of the screen. On medium-sized screens and larger, the width changes to 80%.
+On small screens, a modal is always 100% of the width of the screen. On medium-sized screens and larger, the width changes to 600px. If you're using the Sass version of Foundation, edit the `$reveal-width` variable to change this. This can be a percentage value, or a pixel value.
```
Comment (ncoden) [inline docs/pages/reveal.md]: No need to repeat that. You can simply reference the setting with:
> (see the `$reveal-width` setting).

Comment (DanielRuf) [reply docs/pages/reveal.md]: Done.

File: docs/pages/reveal.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ You can use our handy [close button](close-button.html) styles to do this:
 
 ## Sizing
 
-On small screens, a modal is always 100% of the width of the screen. On medium-sized screens and larger, the width changes to 80%.
+On small screens, a modal is always 100% of the width of the screen. On medium-sized screens and larger, the width changes to 600px (see the $reveal-width setting).
```
Comment (ncoden) [inline docs/pages/reveal.md]: `$reveal-width` with "`"  :smile:

Comment (DanielRuf) [reply docs/pages/reveal.md]: Oops.

Comment (DanielRuf) [reply docs/pages/reveal.md]: Changed =)

**Comment by ncoden** [comment]
👍 LGTM, thx

**Pull request merged**